ethcore/src/machine.rs

```diff
@@ -122,7 +122,13 @@ impl EthereumMachine {
 }
 
 impl EthereumMachine {
-	/// Execute a call as the system address.
+	/// Execute a call as the system address. Block environment information passed to the
+	/// VM is modified to have its gas limit bounded at the upper limit of possible used
+	/// gases including this system call, capped at the maximum value able to be
```

Comment:
I am not sure if I understand this phrase fully. Can you rephrase the part around "…upper limit of possible used gases including this system call"? Thanks!

Reply:
Can you explain more which parts looks confusing? Would help a lot for me to rephrase it.

Reply:
Absolutely. And remember I'm new to much of this so it might just be me being ignorant.  To me it's "upper limit of possible used gases" that is hard to understand.
Maybe something like: "This call executes a call as the system address and modifies the VM's gas limits to be bound to the upper limit, given by the gas amounts provided in the call. Gas is capped to the maximum value represented by a U256" – is that close to what you meant to say?

Reply:
Ok, I think I understand a little better now, thank you for taking the time. I like your suggested change!